data/857/657/75/85765775.geojson

```diff
@@ -138,7 +160,12 @@
         "gp:id":22723689,
         "qs_pg:id":346804
     },
+    "wof:concordances_alt":{
+        "gp:id":22723689,
+        "qs_pg:id":346804
+    },
     "wof:country":"CR",
+    "wof:created":1497313341,
```

Comment:
Be careful of copying `wof:created` over from another record? 

I think generally that's the creation date for _this_ record – versus the superseded record or versus the IRL place (which would be an `edtf` property.

Reply:
Good catch - I have a list of properties to _not_ bring over.. I'll add this one.